conans/test/functional/generators/components/pkg_config_test.py

```diff
@@ -0,0 +1,226 @@
+import os
+import platform
+import textwrap
+import unittest
+
+from nose.plugins.attrib import attr
+
+from conans.client.tools import which, PkgConfig, environment_append
+from conans.model.ref import ConanFileReference
+from conans.test.utils.genconanfile import GenConanfile
+from conans.test.utils.tools import TestClient
+
+
+@unittest.skipUnless(which("pkg-config"), "Requires pkgconfig executable")
```

Comment:
I'd prefer something a bit more explicit, like assuming pkg-config is installed in Linux envs, and fail otherwise. 
Because this has the risk of never running in CI, let's say there is a typo, like the executable is called ``pkgconfig``.

Reply:
Removed so we are able to test this in both windows and linux